build.rs

```diff
@@ -1,15 +0,0 @@
-// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This is a nice cleanup! I would just adjust the commit message to refactor or fix like so:

`refactor: remove redundant crate version variables`

Reply:
done